SPECIFICATION.md

```diff
@@ -0,0 +1,143 @@
+# The FontoBene Font Format, Version 1.0
+
+
+## Introduction
+
+FontoBene is a vector font format. It only uses strokes to form glyphs, in
+order for easy rendering in systems that cannot easily deal with more complex
+parts of formats like TTF or OTF, like Bézier curves.
+
+FontoBene was designed with the use case of text on printed circuit boards
+(PCBs) in mind. PCBs are usually manufactured using the Gerber format as an
+exchange format to describe the etching layers, which does not support
+quadratic / cubic curves.
+
+The main inspiration for this font format was the LFF font used by LibreCAD.
+
+
+## High Level Format Description
+
+Every FontoBene font file consists of UTF-8 encoded text. The lines are
+separated by newlines (U+000A LINE FEED). It starts with a header, followed by
+glyph definitions.
+
+
+### Header
+
+The header is parsed line-by-line, starting with a number sign (`#`) as the
+first character on every line. The number sign is followed by a space, a key,
+and a value. Key and value are separated by a colon (`:`). Everything following
+the colon is considered the value. Both keys and values are strings.
```

Comment:
Should we allow multi-line values? Or should we add a section after the metadata key-value pairs that can be used as free-form text area, e.g. for comments? (See "Notes" [here](https://github.com/LibreCAD/LibreCAD/blob/master/librecad/support/fonts/iso.lff))

Reply:
Maybe use the `#` character for marking lines as comments, like in programming languages.

For metadata, we could use a similar concept as for the glyphs:
```ini
[meta]
version=1.0
license=GPLv3+
...
```

Reply:
I like that :) Will update the spec.